HelloWorld/HelloWorld/Program.cs

```diff
@@ -21,6 +22,9 @@ public double Calculate(double radis)
     public double  CalculateAreaOfRectangle(double length, double width)
     { return length * width; }
 
+    public double Calculatevolumeofcube(double length, double width, double height)
```

Comment:
The name of this function is not using proper title case.  
Please compare to the name of the other calculate function.

Reply:
I have fixed this issue.